src/components/data-management/ProjectDetails.jsx

```diff
@@ -34,6 +43,7 @@ const ProjectDetails = ({ width, height }) => {
     getFromApiExpectOK,
   );
   const [tableData, setTableData] = useState([]);
+  const [tableColumns, setTableColumns] = useState([]);
```

Comment:
Refers to line 63: shouldn't the guard be `activeProject` rather than `projects`?

Reply:
I just checked. There is a brief moment where `activeProject === undefined` when deleting a project. So, `projects` is to be changed here, and additional `else { setSampleNames([]) }` has to be added. Either way is fine, but having `activeProject` as a part of the guard makes the purpose of this block clearer. So, I'll change the guard and add the `else` block.